terraform/environments/delius-core/locals_environments_lower.tf

```diff
@@ -7,6 +7,27 @@ locals {
   db_config_lower_environments = {
     name                 = "db_for_lower_environments"
     ami_name             = "delius_core_ol_8_5_oracle_db_19c_patch_2023-06-12T12-32-07.259Z"
+    instance             = {
+      disable_api_termination       = true
+      instance_type                 = "r6i.xlarge"
+      key_name                      = "key_name_here"
+      metadata_endpoint_enabled     = "enabled"
+      metadata_options_http_tokens  = "required"
+      associate_public_ip_address   = false
+      monitoring                    = true
+      ebs_block_device_inline       = false
+      vpc_security_group_ids        = []
+      private_dns_name_options      = {
+        enable_resource_name_dns_aaaa_record = false
+        enable_resource_name_dns_a_record    = false
+        hostname_type                        = "host_name_type_here"
```

Comment:
Looks to be causing a tf plan error. Would recommend we create a private dns name here, e.g. currently, the DBs in legacy have A records created for say delius-db-1 within the private hosted zone.